docs/_posts/SKocer/2023-05-11-icd10cm_cause_claim_mapper_en.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: model
+title: Mapping ICD10CM Codes with Corresponding Causes and Claim Analysis Codes
+author: John Snow Labs
+name: icd10cm_cause_claim_mapper
+date: 2023-05-11
+tags: [en, licensed, chunk_mapping, icd10cm, cause, claim]
+task: Chunk Mapping
+language: en
+edition: Healthcare NLP 4.4.0
+spark_version: 3.0
+supported: true
+annotator: ChunkMapperModel
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This pretrained model maps ICD10CM codes, subsequently providing corresponding causes and generating claim analysis codes for each respective ICD10CM code.
+
+## Predicted Entities
+
+`icd10cm_cause`, `icd10cm_claim_analysis_code`
+
+{:.btn-box}
+<button class="button button-orange" disabled>Live Demo</button>
+[Open in Colab](https://colab.research.google.com/github/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/Certification_Trainings/Healthcare/26.Chunk_Mapping.ipynb){:.button.button-orange.button-orange-trans.co.button-icon}
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/clinical/models/icd10cm_cause_claim_mapper_en_4.4.0_3.0_1683819210044.zip){:.button.button-orange}
+[Copy S3 URI](s3://auxdata.johnsnowlabs.com/clinical/models/icd10cm_cause_claim_mapper_en_4.4.0_3.0_1683819210044.zip){:.button.button-orange.button-orange-trans.button-icon.button-copy-s3}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+document_assembler = DocumentAssembler()\
+      .setInputCol("text")\
+      .setOutputCol("document")
+
+chunk_assembler = Doc2Chunk()\
+      .setInputCols("document")\
+      .setOutputCol("icd_chunk")
+
+chunkerMapper = ChunkMapperModel.pretrained("icd10cm_cause_claim_mapper", "en", "clinical/models")\
+      .setInputCols(["icd_chunk"])\
+      .setOutputCol("mappings")\
+      .setRels(["icd10cm_cause", "icd10cm_claim_analysis_code"])
+
+pipeline = Pipeline().setStages([document_assembler,
+                                 chunk_assembler,
+                                 chunkerMapper])  
+
+model = pipeline.fit(spark.createDataFrame([['']]).toDF('text')) 
+
+lp = LightPipeline(model)
+
+res = lp.fullAnnotate(["D69.51", "G43.83", "Z83.51"])
+```
+```scala
+val document_assembler = new DocumentAssembler()
+       .setInputCol("text")
+       .setOutputCol("document")
+
+val chunk_assembler = new Doc2Chunk()
+      .setInputCols("document")
+      .setOutputCol("icd_chunk")
+
+val chunkerMapper = ChunkMapperModel
+       .pretrained("icd10cm_cause_claim_mapper", "en", "clinical/models")
```

Comment:
please move this line to the end of ChunkMapperModel

Reply:
Done.